### PR TITLE
docs(MetaOption)

### PR DESCRIPTION
**Pull request by csjkevin**
### PR includes

- [x] documents

### Screenshot

|  Before  |  After  |
|  -  | -  |
| Screen Shot 2021-06-01 at 5 06 48 PM | Screen Shot 2021-06-02 at 10 59 52 AM |

### CODE REVIEW

File: docs/api/options/meta.en.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Input field, definition field.
 
 <description> _(v: any, k?: number) => any_ **optional**</description>
 
-The tick formatting function affects the display of data on Axis, Legend, and Tooltip.
+The tick formatting function, inputs value and value index, returns formatted value, affects the display of data on Axis, Legend, and Tooltip.
```
Comment (visiky) [inline docs/api/options/meta.en.md:65]: 翻译很诡异，不如直接修改类型定义为：_(v: string | number, index?: number) => any_

Comment (csjkevin) [reply docs/api/options/meta.en.md]: OK. 类型定义修改在 https://github.com/antvis/scale/pull/150

File: docs/api/options/meta.zh.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ _MetaOption_ 配置如下：
 | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | 分类度量 | - cat: 分类度量 <br /> - timeCat: 时间分类度量                                                                                                                                                                      |
 | 连续度量 | - linear: 线性度量 <br /> - time：连续的时间度量 <br /> - log: log 度量 <br /> - pow: pow 度量 <br /> - quantize：分段度量，用户可以指定不均匀的分段 <br /> - quantile: 等分度量，根据数据的分布自动计算分段 <br /> |
-| 常量度量 | - identity: 常量度量
```
Comment (hustcc) [inline docs/api/options/meta.zh.md:47]: 无效 diff 可以干掉。

Comment (csjkevin) [reply docs/api/options/meta.zh.md]: 干掉了

File: docs/api/options/meta.en.md (hunk 1 above)
Comment (csjkevin) [reply docs/api/options/meta.en.md]: 类型定义修改已合并

**Pull request merged**